js/controller.js

```diff
@@ -1299,6 +1324,24 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       // In case ad was skipped or errored while stalled
       this.setBufferingState(false);
       this.renderSkin();
+
+      // restore video360 pos by deviceOrientation changes (only for mobile devices)
+      // It's uses after video 360 was moved by device motion, after ads playback
+      if(this.state.deviceOrientation.allow) {
```

Comment:
Could we move this to a separate function? Something like `restoreDeviceOrientation()`. There's a lot of logic in this handler already, so it's a bit hard to reason about the flow